packages/flutter/lib/src/foundation/observer_list.dart

```diff
@@ -43,9 +43,12 @@ class ObserverList<T> extends Iterable<T> {
   ///
   /// Returns whether the item was present in the list.
   bool remove(T item) {
-    _isDirty = true;
-    _set.clear(); // Clear the set so that we don't leak items.
-    return _list.remove(item);
+    final removed = _list.remove(item);
```

Comment:
you forgot the type here, that's the only remaining thing

Reply:
Thanks. Done